### ISSUE TITLE
[BUG]: Taskbar Clock Customization no longer shows time or date properly

### ISSUE THREAD

**Issue by ianTevesAcc** [ramensoftware/windhawk-mods#968]
**Description:**
As described in the title, the mod is longer functioning. I didn't update and I restarted my computer, file explorer and wind hawk, with to no avail. I uninstalled and installed the mod but the same result is reproduced. The date and time is no longer shown properly. You can see this at the end of this bug report.

**Reproduce:**
1. Update Windhawk to latest version
2. Uninstall Taskbar Clock Customization mod
3. Restart Computer
4. Reinstall Taskbar Clock Customization mod
5. Outcome Reproduced - see on screenshots...

**Specs:**
Taskbar Clock Customization Version: 1.3.3
Windhawk Version: 1.5.1

Device: Swift SF314-511
Processor: 11th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz 2.80 GHz
Ram: 16GB
System Type: 64bit, x64based-processor

Windows: 11 Home
Version: 24H2
OS build: 26120.1542
Experience: Windows Feature Experience Pack 1000.26100.18.0

**Screenshots**
Expected Outcome:
![image](https://github.com/user-attachments/assets/62c4a9c9-0f4a-4cfc-a5d2-c892fd9af0f1)

Actual Outcome:
![image](https://github.com/user-attachments/assets/5a2ccfc1-0150-42c2-b5af-e6c3b88c9384)

**Additional Information:**
Thank you for the work on this mod. I've been loving it, and would love to continue using it in the future after this bug has been resolved.
Btw this is the mod settings I use normally, in case it helps.
```
{"ShowSeconds":1,"TimeFormat":"h':'mm':'ss tt","DateFormat":"ddd',' MMM dd yyyy","WeekdayFormat":"dddd","TopLine":"%time%","BottomLine":"%date%","MiddleLine":"%weekday%","TooltipLine":"%web1_full%","Width":180,"Height":60,"TextSpacing":0,"WebContentsItems[0].Url":"https://feeds.bbci.co.uk/news/world/rss.xml","WebContentsItems[0].BlockStart":"<item>","WebContentsItems[0].Start":"<title><![CDATA[","WebContentsItems[0].End":"]]></title>","WebContentsItems[0].MaxLength":28,"WebContentsUpdateInterval":10,"TimeStyle.Visible":1,"TimeStyle.TextColor":"","TimeStyle.TextAlignment":"","TimeStyle.FontSize":0,"TimeStyle.FontFamily":"","TimeStyle.FontWeight":"","TimeStyle.FontStyle":"","TimeStyle.FontStretch":"","TimeStyle.CharacterSpacing":0,"DateStyle.TextColor":"","DateStyle.TextAlignment":"","DateStyle.FontSize":0,"DateStyle.FontFamily":"","DateStyle.FontWeight":"","DateStyle.FontStyle":"","DateStyle.FontStretch":"","DateStyle.CharacterSpacing":0,"oldTaskbarOnWin11":0}
```

But the bug is still produced without using my personal settings config.

@m417z

**Comment by m417z** [comment]
I'm using this mod on multiple computers and I never experienced this issue. Are you using any other customization tools?

Please attach debug logs by following the instructions on the Troubleshooting page:
https://github.com/ramensoftware/windhawk/wiki/Troubleshooting#some-or-all-windhawk-mods-dont-work

> * Disable the mod.
> * Go to the mod's Advanced tab.
> * Set **Debug logging** to **Detailed debug logs**.
> * Click on **Show log output**.
> * Enable the mod.
> * Verify that the mod doesn't work, copy the log and save it to a file.
> * Set **Debug logging** back to **None**.

**Comment by JergenRoss** [comment]
same me, this appeared to happen for me after updating Windows (26100.2448)

I found a bug, it caused toggle abbreviated time and date from Windows settings


https://github.com/user-attachments/assets/fd919675-0fa5-4816-a5c9-0906471e666c

I think abbreviates affect something else `%variable%` or format like `%time%`, `%date%`, `ddd`, etc.

**Comment by m417z** [comment]
@JergenRoss does it work correctly when the abbreviated time and date option is disabled?
I'll look at it when I have time, but currently it only affects preview builds so it's not a priority.

**Comment by JergenRoss** [comment]
Yes correct, Of course. if the regular version is released, this should fix the bug.

**Comment by ianTevesAcc** [comment]
At the time I got this bug while I was using the Windows Developer Build preview so that might be why I experienced some bugs with it.

**Comment by MatthewHxlden** [comment]
> At the time I got this bug while I was using the Windows Developer Build preview so that might be why I experienced some bugs with it.

It is cos Windows update by default changed a setting.

Go to settings>>time & language>>date & time>> click the "show time and date in system tray" dropdown menu, deselect "show abbreviated time" 

this brought mine all back with the windhawk mod enabled

**Comment by m417z** [comment]
Fixed in version 1.4, please let me know if you encounter any other issues with the new "Show abbreviated time and date" option.
